python/dask_cudf/dask_cudf/tests/test_core.py

```diff
@@ -135,11 +136,26 @@ def test_set_index(nelem):
         dgdf = ddf.map_partitions(cudf.from_pandas)
 
         expect = ddf.set_index("x")
-        got = dgdf.set_index("x")
+        got = dgdf.set_index("x", divisions=divisions)
 
         dd.assert_eq(expect, got, check_index=False, check_divisions=False)
 
 
+@pytest.mark.parametrize("by", ["a", "b"])
+@pytest.mark.parametrize("nelem", [10, 500])
+@pytest.mark.parametrize("nparts", [1, 10])
+def test_set_index_quantile(nelem, nparts, by):
+    df = cudf.DataFrame()
+    df["a"] = np.ascontiguousarray(np.arange(nelem)[::-1])
+    df["b"] = np.random.choice(cudf.datasets.names, size=nelem)
+    ddf = dd.from_pandas(df, npartitions=nparts)
+
+    with dask.config.set(scheduler="single-threaded"):
```

Comment:
Is this line left over from debugging or is this necessary for the test ?

Reply:
The schduler is explicitly set to `"single-threaded"` for a few of the `dask_cudf` tests.  I'm not really sure if there was a reason for this.  I removed the line here, but it may also make sense to remove the others..

Reply:
I suspect that this is a holdover from a long time ago when I was doing things.  There may have been some thread-safety issues early on?